caffe2/CMakeLists.txt

```diff
@@ -169,7 +182,7 @@ if (BUILD_TEST)
           ${test_name} ${Caffe2_MAIN_LIBS} ${Caffe2_DEPENDENCY_LIBS}
           gtest_main)
     endif()
-    if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.0)
+    if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.0 AND NOT ANDROID)
```

Comment:
Why the change? The sources still use `cxx_range_for` when building for android. `target_compile_features` tells CMake to set appropriate flags. Without it, compiler may default to c++98.

Reply:
`target_compile_features` is not recognized before for some reason.. resolved now.